azure-pipelines/release.yml

```diff
@@ -55,6 +55,10 @@ stages:
         parameters:
           ubuntu_version: 20.04
           job_name: 'Ubuntu20_04'
+      - template: jobs/release_ubuntu.yml
+        parameters:
+          ubuntu_version: 22.04
+          job_name: 'Ubuntu22_04'
```

Comment:
Need to override the gcc version here as well.

Reply:
Yes, I noticed that. Still working on it. Is it reasonable to use `gcc_version` 10 for Debian and Ubuntu 22.04 and `gcc_version` 8 for all older Ubuntu versions (like before)?